runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordGameEventType.java

```diff
@@ -262,7 +294,95 @@
+	RAIDS_THEATRE_OF_BLOOD("Theatre of Blood", DiscordAreaType.RAIDS, Varbits.THEATRE_OF_BLOOD),
+
+	// Other
+	REGION_ABYSSAL_AREA("Abyssal Area", DiscordAreaType.REGIONS, 12108),
+	REGION_ABYSSAL_NEXUS("Abyssal Nexus", DiscordAreaType.REGIONS, 12106),
+	REGION_AGILITY_PYRAMID("Agility Pyramid", DiscordAreaType.REGIONS, 13356),
+	REGION_AIR_ALTAR("Air Altar", DiscordAreaType.REGIONS, 11339),
+	REGION_AL_KHARID_MINE("Al Kharid Mine", DiscordAreaType.REGIONS, 13107),
+	REGION_APE_ATOLL("Ape Atoll" , DiscordAreaType.REGIONS, 10795, 10974, 11050),
+	REGION_ARANDAR("Arandar", DiscordAreaType.REGIONS, 9266, 9267, 9523),
+	REGION_ASGARNIA("Asgarnia", DiscordAreaType.REGIONS, 11825, 11829, 11830, 12085, 12086),
+	REGION_BATTLEFIELD("Battlefield", DiscordAreaType.REGIONS, 10034),
+	REGION_BATTLEFRONT("Battlefront", DiscordAreaType.REGIONS, 5433, 5434),
+	REGION_BLAST_MINE("Blast Mine", DiscordAreaType.REGIONS, 5948),
+	REGION_BODY_ALTAR("Body Altar", DiscordAreaType.REGIONS, 10059),
+	REGION_CHAOS_ALTAR("Chaos Altar", DiscordAreaType.REGIONS, 9035),
+	REGION_COSMIC_ALTAR("Cosmic Altar", DiscordAreaType.REGIONS, 8523),
+	REGION_COSMIC_ENTITYS_PLANE("Cosmic Entity's Plane", DiscordAreaType.REGIONS, 8267),
+	REGION_CRABCLAW_ISLE("Crabclaw Isle", DiscordAreaType.REGIONS, 6965),
+	REGION_CRAFTING_GUILD("Crafting Guild", DiscordAreaType.REGIONS, 11571),
+	REGION_CRANDOR("Crandor", DiscordAreaType.REGIONS, 11314, 11315),
+	REGION_CRASH_ISLAND("Crash Island", DiscordAreaType.REGIONS, 11562),
+	REGION_DARK_ALTAR("Dark Altar", DiscordAreaType.REGIONS, 6716),
+	REGION_DEATH_ALTAR("Death Altar", DiscordAreaType.REGIONS, 8779),
+	REGION_DEATH_PLATEAU("Death Plateau", DiscordAreaType.REGIONS, 11320),
+	REGION_DENSE_ESSENCE("Dense Essence Mine", DiscordAreaType.REGIONS, 6972),
+	REGION_DRAGONTOOTH("Dragontooth Island", DiscordAreaType.REGIONS, 15159),
+	REGION_EAGLES_PEAK("Eagles' Peak", DiscordAreaType.REGIONS, 9270),
+	REGION_EARTH_ALTAR("Earth Altar", DiscordAreaType.REGIONS, 10571),
+	REGION_ENCHANTED_VALLEY("Enchanted Valley", DiscordAreaType.REGIONS, 12102),
+	REGION_FALADOR_FARM("Falador Farm", DiscordAreaType.REGIONS, 12083),
+	REGION_FARMING_GUILD("Farming Guild", DiscordAreaType.REGIONS, 4922),
+	REGION_FELDIP_HILLS("Feldip Hills", DiscordAreaType.REGIONS, 9773, 9774, 10029, 10030, 10285, 10286, 10287, 10542),
+	REGION_FIRE_ALTAR("Fire Altar", DiscordAreaType.REGIONS, 10315),
+	REGION_FISHER_REALM("Fisher Realm", DiscordAreaType.REGIONS, 10569),
+	REGION_FISHING_GUILD("Fishing Guild", DiscordAreaType.REGIONS, 10293),
+	REGION_FISHING_PLATFORM("Fishing Platform", DiscordAreaType.REGIONS, 11509),
+	REGION_FORSAKEN_TOWER("The Forsaken Tower", DiscordAreaType.REGIONS, 5435),
+	REGION_FOSSIL_ISLAND("Fossil Island", DiscordAreaType.REGIONS, 14394, 14395, 14649, 14650, 14651, 14652, 14906, 14907, 14908, 15162, 15163, 15164),
+	REGION_FREMENNIK("Fremennik Province", DiscordAreaType.REGIONS, 10296, 10552, 10808, 10809, 11064, 10810, 10811),
```

Comment:
Are we ordering regions in numerical order?
```suggestion
	REGION_FREMENNIK("Fremennik Province", DiscordAreaType.REGIONS, 10296, 10552, 10808, 10809, 10810, 10811, 11064),
```